src/commands/debug_files/mod.rs

```diff
@@ -3,6 +3,7 @@ use clap::{ArgMatches, Command};
 
 pub mod bundle_sources;
 pub mod check;
+pub mod create_jvm_based_bundle;
```

Comment:
This command is quite a mouthful. What about `sentry-cli debug-files bundle-jvm` or similar? I have no hard feeling about it, but typing `create-jvm-based-bundle` is "a lot" :P